playground/backend/internal/setup_tools/builder/setup_builder_test.go

```diff
@@ -30,6 +30,9 @@ func TestSetupExecutor(t *testing.T) {
 	pipelineId := uuid.New()
 	sdk := pb.Sdk_SDK_JAVA
 	lc, err := fs_tool.NewLifeCycle(sdk, pipelineId, "")
+	if err != nil {
+		panic(err)
```

Comment:
could we use t.error instead? 
```suggestion
		t.Error(err)
```

Reply:
Changed.